src/api/app/components/bs_request_action_description_component.rb

```diff
@@ -0,0 +1,55 @@
+# This component renders the request action description based on the type of the action
+
+class BsRequestActionDescriptionComponent < ApplicationComponent
```

Comment:
Maybe this component should go into it's own PR?

Reply:
The problem is that without this component the full code-path doesn't work because only the beta version is using it and it starts to use it at the time the switch from the `Hash` to the `BsRequestAction` happens.

A separate PR to refactor the component and get rid of the `html_safe` will happen for sure though

Reply:
Then why don't pass first the component using the old hash version, and then switch it to the OOP version?

Reply:
https://github.com/openSUSE/open-build-service/pull/15817